src/js/components/Login.vue

```diff
@@ -1,33 +1,31 @@
 // TODO: check if the session is valid before trying to login with locatdata
 <template>
-  <div class="col col-6 push-middle push-center login-wrapper">
-    <div class="header"><h2>Login</h2></div>
-    <form id="login" class="form left" @submit.prevent="onSubmit">
-        <div class="error" v-show='errMsg.length'>
-          {{ errMsg }}
-        </div>
-        <div class="form-item">
-          <label>User name <span class="req"></span> <span class="error">{{ usernameErr }}</span></label>
-          <input type="text" placeholder="Your login name" class="small" :class='{"error": usernameErr.length}' v-model='user'>
-        </div>
-        <div class="form-item">
-          <label>Password <span class="req"></span> <span class="error">{{ passwordErr }}</span></label>
-          <input type="password" placeholder="Your password" class="small" :class='{"error": passwordErr.length}' v-model='password'>
-        </div>
-        <div class="form-item">
-          <label>Language / Sprache / שפה</label>
-          <select v-model="language">
-            <option value="en">English</option>
-            <option value="de">Deutsche</option>
-            <option value="hb">עברית</option>
-          </select>
-        </div>
-        <div class="form-item">
-          <button type="submit">Submit</button>
-          <a href="mailto:martin.schroeter@theologie.uni-goettingen.de?subject=Forgot%20password">Forgot my login</a>
-        </div>
-    </form>
-  </div>
+  <el-card class="loginCard" header="Login" :body-style="{ background: 'white' }">
+    <el-form id="login" @submit.native.prevent="onSubmit" :label-position="labelPosition">
+      <el-form-item class="error warningText" v-show='errMsg.length'>
+        {{ errMsg }}
+      </el-form-item>
+      <el-form-item>
+        <span slot="label">User name <span class="reqired warningText"></span> <span class="error warningText">{{ usernameErr }}</span></span>
+        <el-input class="loginUsername" type="text" placeholder="Your login name" :class='{"error": usernameErr.length}' v-model='user'></el-input>
+      </el-form-item>
+      <el-form-item>
+        <span slot="label">Password <span class="reqired warningText"></span> <span class="error warningText">{{ passwordErr }}</span></span>
+        <el-input class="loginPassword" type="password" placeholder="Your password" :class='{"error": passwordErr.length}' v-model='password'></el-input>
+      </el-form-item>
+      <el-form-item label="Language / Sprache / שפה">
+        <el-select class="langSelect" v-model="language">
+          <el-option value="en">English</el-option>
+          <el-option value="de">Deutsche</el-option>
+          <el-option value="hb">עברית</el-option>
+        </el-select>
+      </el-form-item>
+      <el-form-item>
+        <el-button type="primary" native-type="submit">Submit</el-button>
+        <a href="mailto:Bronson.Brown-deVost@theologie.uni-goettingen.de?subject=Forgot%20password">Forgot my login</a>
```

Comment:
We will need a more rigorous forgotten password pathway at some point when the system goes to production.